src/jit/unwind.cpp

```diff
@@ -153,7 +157,12 @@ void Compiler::unwindPushPopCFI(regNumber reg)
         // since it is pushed as a frame register.
         || (reg == REG_FPBASE)
 #endif // ETW_EBP_FRAMED
-#endif
+#endif // UNIX_AMD64_ABI
+#if defined(_TARGET_UNIX_)
```

Comment:
I have left not comment on the original change that this `#if defined(_TARGET_UNIX_)` is not necessary (this whole function is under Unix ifdef already). Not sure whether you have missed it or whether you kept it intentionally.

Reply:
@jkotas I've tried it but  I have OSX testing fails without it.

Reply:
It does not make sense for OSX testing to fail because of ARM-specific code. Was it just a unrelated intermittent failure or infrastructure problem?